src/tools/rpgcombobox.h

```diff
@@ -0,0 +1,63 @@
+#ifndef RPGCOMBOBOX_H
+#define RPGCOMBOBOX_H
+
+#include <QObject>
+#include <QComboBox>
+#include <QAbstractItemModel>
+#include <QCompleter>
+#include <functional>
+#include "rpgmodel.h"
+#include "../dialogedit.h"
+
+
+template <class MODEL>
```

Comment:
Here you can pass any class, that is dereivec from QAbstractItemModel, in this case I pass the RpgModels I declared before to use them as a model for the combobox to make it easier to populate all comboboxes with data.